cookbook/product/query-products.rst

```diff
@@ -13,7 +13,7 @@ In Akeneo PIM, products can be stored and accessed through Doctrine ORM (EAV lik
 
 The PQB aims to abstract the used persistence storage to provide the same operations in both cases.
 
-Instantiate a new product query builder
+Instantiate a New Product Query Builder
```

Comment:
Uppercase at each word is an accepted usage for title. So uppercase everything or only the first word.